iib/workers/dogpile_cache.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import functools
+
+from dogpile.cache import make_region
+
+from iib.workers.config import get_worker_config
+
+
+def skopeo_inspect_should_use_cache(*args, **kwargs):
+    """Return true in case this requests can be taken from or stored in cache."""
+    return any([arg.find('@sha256:') != -1 for arg in args])
```

Comment:
It'd be more efficient to use a generator rather than creating a list. Then you don't have to evaluate all the args.
```suggestion
    return any(arg.find('@sha256:') != -1 for arg in args)
```

Reply:
Agree